source/Playnite/GamesEditor.cs

```diff
@@ -1386,8 +1416,14 @@ private void Controllers_Stopped(object sender, GameStoppedEventArgs args)
                 Application.Discord?.ClearPresence();
             }
 
+            if (wasHdrEnabled.HasValue)

```

Comment:
1) This will always have value after first game is started, so I don't think it makes sense to have it nullable.
2) What will happen if multiple games are started? Each with different HDR options.

Reply:
Regarding 2, my feature basically works in the following ways:

Case 1: Game's HDR toggle is on and system HDR is currently off. Then upon launching the game, system HDR is turned on. And upon exiting the game system, HDR is turned off

Case 2: Game's HDR toggle is on and system HDR is currently on. Then upon launching the game, nothing new happens. And upon exiting the game, nothing new happens (system HDR continues to be on)

Case 3: Game's HDR toggle is off and system HDR is either on or off (doesnt matter). Upon entering the game, nothing happens, and upon exiting the game nothing happens. Basically, if the game has HDR set to disabled, then the game functions the same as if I had never written this feature.



Taking the above into account, if enable HDR for game A and have it disabled for game B and then launch both of them, then HDR will only be disabled once game A exits.

There is however a corner case that you've made me think of where if HDR is set on for both game A and B. If you launch A followed by B and then kill A, it will most likely disable system HDR even though game B still wants it. But I don't think this would happen if you launched A then B then killed B (see case 2 above)

Reply:
Point number 1 has been addressed. I need to think more about point number 2 still

Reply:
Point number 2 is now addressed as well. Essentially, I will only get the system's current HDR state if there are no games requiring HDR that are currently playing. And I will only reset the system's HDR state back to its original state once there are no more games being played that require HDR.